### ISSUE TITLE
Do you have a discord I can connect to, or a slack channel?

### ISSUE THREAD

**Issue by aweb3dad** [eddiemay/DD4-Biblical#1]
Asking so we can talk at a faster pace. Unless it's okay if I use this platform to have conversation on.. which I'm okay with. Had some questions.

I noticed some ng-* on the html... is the frontend made with angular 1? Also what's the best way to contribute? Any if I push code to here is there a place I would be able to see it? Like via aws, or any other platform. So curious about the app here and it's architecture and how you view and test your changes.

**Comment by eddiemay** [comment]
Yes, it is an angular JS project which is angular 1.5. I never got that into angular 2.0 I will probably update it to that one day when I redo the UI but for now I had a lot of angularJS built out so used that.
